mkinitcpio

```diff
@@ -251,6 +260,79 @@ build_image() {
     fi
 }
 
+build_uefi(){
+    local out=$1 initramfs=$2 cmdline=$3 osrelease=$4 splash=$5 kernelimg=$6 uefistub=$7 microcode=(${@:7}) errmsg=
+    OBJCOPYARGS=()
+
+    msg "Creating UEFI executable: %s" "$out"
+
+    if [[ -z "$uefistub" ]]; then
+        for stub in {/usr,}/lib/{systemd/boot/efi,gummiboot}/linux{x64,ia32}.efi.stub; do
```

Comment:
the list this generates is:
```
/usr/lib/systemd/boot/efi/linuxx64.efi.stub
/usr/lib/systemd/boot/efi/linuxia32.efi.stub
/usr/lib/gummiboot/linuxx64.efi.stub
/usr/lib/gummiboot/linuxia32.efi.stub
/lib/systemd/boot/efi/linuxx64.efi.stub
/lib/systemd/boot/efi/linuxia32.efi.stub
/lib/gummiboot/linuxx64.efi.stub
/lib/gummiboot/linuxia32.efi.stub
```
just to make sure that's the intended behavior

Reply:
Yep, that seems correct. It should support !usrmerge, and any distros that doesn't ship systemd.